app/assets/stylesheets/style.scss

```diff
@@ -0,0 +1,145 @@
+/*font-family: 'Courgette', cursive;
+font-family: 'Amaranth', sans-serif;
+font-family: 'Nothing You Could Do', cursive;
+font-family: 'Berkshire Swash', cursive;*/
+$main-font: 'Berkshire Swash', cursive;
+$fallback: #cccccc;
+$home-font: #d68533;
+$home-bttn: #a32900;
+$home-bttn-hover: #621900;
+$grain: #ebc633;
+$grain-hover: #e6b800;
+$hop: #5d8d5d;
+$hop-hover: #547f54;
+$yeast: #C1693C;
+$yeast-hover: #AE5E36;
+
+
+
+// General Styling   //
+body {
+  
+}
+
+.background-image {
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: -1;
+
+  display: block;
+  background-image: image-url("HD-Wallpaper-Cat-With-Beer.jpeg");
+  background-color: $fallback;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+}
+
+// Home page styling //
+
+.font-pretty {
+  font-family: $main-font;
+  color: $home-font;
+  font-size: 450%;
+
+  &.smaller {
+    font-size: 215%;
+    padding-left: 2rem;
+  }
+
+}
+
+.recipe-maker-button {
+  
+  .button {
+    background-color: #A32900;
+    margin: 3rem;
+  }
+
+  .button:hover {
+    background-color: #621900;
+  }
+}
+
+.login-to-use {
+  
+  .button {
+    background-color: #A32900;
+    margin: 3rem;
+  }
+
+  .button:hover {
+    background-color: #621900;
```

Comment:
Color literals like `#621900` should only be used in variable declarations; they should be referred to via variable everywhere else.